packages/client/src/Publisher.ts

```diff
@@ -147,6 +147,9 @@ export default class BrubeckPublisher implements Context, Stoppable {
         count?: number
         messageMatchFn?: (msgTarget: StreamMessage, msgGot: StreamMessage) => boolean
     } = {}) {
+        if (!streamMessage || !streamMessage.spid) {
```

Comment:
As `streamMessage: StreamMessage` is not an optional parameter, it should be caller's responsibility to ensure that the method is called with a valid value. The method should be able to rely on the contract it defined in the parameter list definition.